### DISCUSSION TITLE
Dashboard shows other currencies without converting?

### DISCUSSION THREAD

**Discussion by iisharankov** [firefly-iii/firefly-iii#6933] [General]
Hello,

So I mainly use CHF/Euro since I work/live in both, and never noticed this until I converted some money to SEK to transfer to a swedish friend. I added the SEK currency (to my knowledge correctly)
![image](https://user-images.githubusercontent.com/30711273/214812751-986d912f-6318-416f-bd3e-02aeaa4bb461.png)
and moved 180 euro -> 2000 SEK into the account. 
![image](https://user-images.githubusercontent.com/30711273/214814235-7e7ff1bb-0bb1-4ac2-ad41-d8eb72a068f8.png)

The thing is, when I go to my dashboard, it shows on the right in euro (great, it's my default and I'm happy if the dashboard shows all accounts/currencies in Euro), but the line for SEK is at 2000. Hovering over it clearly states the value is at 2000 SEK, but on the dashboard that's translated to 2000 Euro, clearly false and very misleading. 

I'm wondering why the dashboard (and elsewhere) does not convert this to euro for representation, and if this can be done/is my setup wrong somehow? Seen below:
![image](https://user-images.githubusercontent.com/30711273/214813380-d26f1be4-d40b-401b-81f5-09d61b6ba8c0.png)

I didn't notice this before when I only had Euro/CHF since they're almost 1 to 1 now, but clearly there's an issue. God forbid I transfer 1 Euro to VEF as then it'll look like I'm a millionaire!

**Comment by JC5** [comment]
This has been discussed a few times already and is a side effect of the current layout. I hope to rectify this in the new layout but there is no ETA yet.
